docs/core-features/hosting.md

```diff
@@ -5,7 +5,7 @@ sidebar_position: 200
 
 ## Self-Hosted
 
-`zrok` is not limited to a managed offering. You can host your own version of `zrok` as well. `zrok` is
+`zrok` is not limited to a managed offering. You can host your own version of `zrok` as well and there's [a self-hosting guide](../guides/v0.3_self_hosting_guide.md). `zrok` is

```

Comment:
how about just changing "host your own" to "[host your own](../guides/v0.3_self_hosting_guide.md)" instead?